projects/igniteui-angular/src/lib/grids/filtering/base/grid-filtering-row.component.ts

```diff
@@ -482,6 +482,14 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
         this.value = value;
     }
 
+    /** @hidden @internal */
+    public inputGroupPrefixClick(event: MouseEvent) {
+        event.preventDefault();
+        event.stopPropagation();
+        (event.currentTarget as any).focus();
```

Comment:
Also, not any, `HTMLElement` should do :)

Reply:
fixed